src/main/java/de/tum/in/www1/artemis/domain/GradingScale.java

```diff
@@ -155,6 +172,34 @@ GradeStep maxGrade() {
         return getGradeSteps().stream().filter(gradeStep -> gradeStep.isUpperBoundInclusive() && gradeStep.getUpperBoundPercentage() == 100.0).findAny().orElse(null);
     }
 
+    public String getPlagiarismGrade() {
+        return plagiarismGrade;
+    }
+
+    public void setPlagiarismGrade(String plagiarismGrade) {
+        this.plagiarismGrade = plagiarismGrade;
+    }
+
+    @JsonIgnore
+    @Nonnull
+    public String getPlagiarismGradeOrDefault() {
+        return plagiarismGrade != null ? plagiarismGrade : DEFAULT_PLAGIARISM_GRADE;
```

Comment:
Same for `getNoParticipationGradeOrDefault`  below.
```suggestion
        return Objects.requireNonNullElse(plagiarismGrade, DEFAULT_PLAGIARISM_GRADE);
```